### ISSUE TITLE
Feature: Add flag to run one command and exit right away

### ISSUE THREAD

**Issue by ratberryjam** [mischah/itunes-remote#9]
It would be nice to have a flag when running itunes-remote to tell it to just perform the specified command and exit, say if you just want to quickly pause or stop playback.

**Comment by mischah** [comment]
@ratberryjam 

Just to make sure I get it straight.

You man instead of:

``` bash
$ itunes-remote 
iTunes: play
Hold on …
✔ Playing ♪♬
iTunes: exit --force
$ 
```

Something like:

``` bash
$ itunes-remote 
iTunes: play --exit
Hold on …
✔ Playing ♪♬
$ 
```

This is definitely a reasonable use case :ok_hand:

**Comment by ratberryjam** [comment]
Exactly 👍

**Comment by al-the-x** [comment]
Additionally:

```
$> itunes-remote play --exit
Hold on …
✔ Playing ♪♬
$>
```

But I assume that providing `--exit` to `play` would support both use cases? 

I WILL SUBMIT A PR **TODAY** FOR THIS.

And immediately `alias play='itunes-remote play --exit'`...

**Comment by jamesratcliffe** [comment]
@al-the-x Ideally it would work for play, stop, pause, next, previous and back

**Comment by quentindemetz** [comment]
Would also like to see this :)

**Comment by odedpriva** [comment]
would be great !

**Comment by mischah** [comment]
Implemented for the following commands: `play`, `stop`, `pause`, `next`, `previous` and `back`.

Example usage from the interactive mode:

```
$ itunes
iTunes: start --exit
$
```

Example usage from outside:

```
$ itunes stop --exit
iTunes:
✔ Stopped playing ♪♬
$
```

**Comment by mischah** [comment]
Published with [2.3.0](https://github.com/mischah/itunes-remote/releases/tag/2.3.0) 🎉
